.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -11,4 +11,11 @@ Add as many details as possible to help others reproduce the issue and test the
 
 *
 
-Fixes #
+<!--
+Link a related issue to this PR. If the PR does not immediately resolve the issue,
+for example requires a separate deployment to be rolled out to production, avoid
```

Comment:
This is a nice nudge I think.

I've nevertheless just been possessed by a pedant demon, who has coerced me into raising the suggestion that we use a linking verb in this sentence:

```suggestion
for example, it requires a separate deployment to be rolled out to production, avoid
```

You may ignore this demon.

Another idea for later might be to add a separate PR template, e.g., one for PRs that immediately "fix an issue" and therefore use the "Fixes" keyword. 

https://docs.github.com/en/github/building-a-strong-community/configuring-issue-templates-for-your-repository#configuring-the-template-chooser

We could then customize the content and labels accordingly if required.

🤷

Reply:
Nice, I like it!